src/components/Title/Title.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface TitleProps {
+  title: React.ReactNode
+  link?: string
+}
+
+export const Title = ({ title, link }: TitleProps): React.ReactElement => {
+  const classes = classnames('usa-logo')
+
+  return (
+    <div className={classes}>
+      <em className="usa-logo__text">
+        <a href={link}>{title}</a>
```

Comment:
A couple things. 
- 🤔 This very much feels like it should be a `<h1>`.  It is literally our site title- for accessibility reasons too.  I wonder why `em` is being used by the uswds design team...
-  As written, `link` prop could be undefined - we should probably not wrap the text in an anchor tag in the first place for those cases.